src/openforms/formio/api/validators.py

```diff
@@ -97,3 +103,54 @@ def __call__(self, value: UploadedFile) -> None:
                     filename=value.name, file_type=f".{ext}"
                 )
             )
+
+
+class NoVirusValidator:
+    def __call__(self, uploaded_file: UploadedFile) -> None:
+        config = GlobalConfiguration.get_solo()
+        if not config.enable_virus_scan:
+            return
+
+        scanner = clamd.ClamdNetworkSocket(
+            host=config.clamav_host,
+            port=config.clamav_port,
+            timeout=config.clamav_timeout,
+        )
+
+        uploaded_file.file.seek(0)
+
+        try:
+            result = scanner.instream(uploaded_file.file)
+        except Exception as exc:
+            logger.error("ClamAV error: %s", exc.args[0])
```

Comment:
please don't remove the exception information itself from the logging call - we want this to be visible in Sentry.

```suggestion
            logger.error("ClamAV error", exc_info=exc)
```

or 
```suggestion
            logger.exception("ClamAV error")
```